app/views/content/how-to-apply-for-teacher-training/teacher-training-personal-statement.md

```diff
@@ -57,7 +57,16 @@ You could include:
 
 Teacher training providers want to see your passion and that you understand the bigger picture of teaching.
 
-Make sure you check your spelling and grammar in your application. You want to make the best possible impression.
+### How to write your personal statement
+
+When writing your personal statement you should make sure you check your spelling and grammar in your application. You want to make the best possible impression.
+
+You can use ChatGPT or other artificial intelligence (AI) tools to help you write your personal statement. You should not rely on it to write your entire statement because:
+
+* AI tools use bland language and will not be able to give details about you as a person and is likely to result in your application being unsuccessful
```

Comment:
The last part of this sentence doesn't quite make sense `.... is likely to result in your application being unsuccessful`

Should it be split into two sentences within this bullet?

@caitlin-reidy @MaxineCl

Reply:
I think it's just missing and **using them** is... but it could be split into 2 as it's a long sentence.

Reply:
have updated this text now

Reply:
definitely reads better now but I think it could probably still be split into two as it feels a bit convoluted with two `and`s?

Reply:
I've made it into 2 sentences now